src/main/resources/application.yml

```diff
@@ -13,6 +15,8 @@ spring:
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
 
+  mvc.pathmatch.matching-strategy: ant_path_matcher
```

Comment:
Needed for springfox issues with SpringBoot 2.5+

Reply:
Comment with to-be-binned with "PUD-1231"?